lib/roda/form_builder/builder.rb

```diff
@@ -0,0 +1,47 @@
+module Roda
+  module FormBuilder
+    module Builder
+      include ApplicationHelper
+      include GOVUKDesignSystemFormBuilder::Builder
+
+      def with_guidance(attribute_name, html)
+        return if html.nil?
+
+        guidance_url = GuidanceUrl.new(@object_name, attribute_name).to_s
+
+        if guidance_url.present?
+          doc = Nokogiri::HTML.fragment(html)
+
+          doc.xpath("div").first.add_child(content_tag(:p, class: "guidance") {
```

Comment:
I think this is good for now. Wondering if we should add ARIA rules to make sure that the guidance hint is associated correctly with the form. Maybe overriding the block (see [injecting content](https://govuk-form-builder.netlify.app/building-blocks/injecting-content/))?

Reply:
I had a crack at this, but the way `&block` works injects content before the field, not after, which means we'll have to dig into quite a bit of the code to make this work. Might be nice as an improvement for the upstream gem though